tests/projects/local-kit/kits/core-kit/Cargo.toml

```diff
@@ -9,14 +9,11 @@ exclude = ["README.md"]
 
 [package.metadata.build-kit]
 included-packages = [
-    # in-tree packages
-    "hello-agent",
-    "hello-go",
+    "pkg-a",
```

Comment:
Are we going to allow users to build a kit from segments of another kit (As in a user wants everything in core kit cept the kernel which they want to add their own). If so this should probably be a table of some sort so that version requirements could be specified.

Reply:
We aren't considering adding versioning at the level of package name right now. The versioning will take place at the level of kit metadata (we think).